src/backend/app.js

```diff
@@ -21,10 +23,82 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+// app.get("/my-route", (req, res) => {
+//   res.send("Hi friend");
+// });
+
+// /future-meals	Respond with all meals in the future (relative to the when datetime)
+
+app.get("/future-meals", async (req, res) => {
+  try {
+    const rows = await knex.raw("SELECT * FROM `meal` WHERE `when` >= NOW()");
+    rows[0].length === 0
+      ? res.status(200).send(rows.slice(0, 1))
+      : res.send(rows.slice(0, 1));
```

Comment:
I dont understand the point of return rows.slice(0, 1). Could you please elaborate?
I would simply return rows[0] as response.